tests/apps/actordotnet/CarActor.cs

```diff
@@ -0,0 +1,36 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace DaprDemoActor
+{
+    using Dapr.Actors.Runtime;
+    using System.Threading.Tasks;
+    using System.Text.Json;
+
+    [Actor(TypeName = "DotNetCarActor")]
+    public class CarActor : Actor, ICarActor
+    {
+        private int count = 0;
+
+        public CarActor(ActorHost host) : base(host)
+        {
+        }
+
+        public Task<int> IncrementAndGetAsync(int delta) {
```

Comment:
If you're going to format it Java how will we tell that it's C#? :trollface:

Reply:
Let me fix the formatting. Working on this required me to code in 4 languages.